### PR TITLE
Assignment A03 completed

### PR DESCRIPTION
**Pull request by Ajay07Metamation**

### CODE REVIEW

File: Training/Program.cs [hunk 1]
```diff
@@ -1,7 +1,26 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿//This program is spelling bee game
+using static System.Console;
+//Importing the words.txt file  
+string[] words = File.ReadAllLines ("D:\\OneDrive - Trumpf Metamation Pvt Ltd\\Desktop\\ASSIGNMENTS\\words.txt");
```
Comment (rajeevramanTMM) [inline Training/Program.cs:4]: Your program crashes if the mentioned file does not exists.

File: Training/Program.cs [hunk 2]
```diff
@@ -1,7 +1,26 @@
...
+char[] letters = new[] { 'U', 'X', 'A', 'L', 'T', 'N', 'E' };
+int total = 0;
+// USing LINQ operators to filter and order the words 
+foreach(var data in words.Where(isValid)
+                         .Select(getscore)
+                         .OrderByDescending(a =>a.score)
+                         .ThenBy(a => a.word)) {
+   if (data.pangram) ForegroundColor = ConsoleColor.Green;
+   Console.WriteLine ($"{data.score,4} {data.word}");
```
Comment (rajeevramanTMM) [inline Training/Program.cs:13]: Remove unused console.

File: Training/Program.cs [hunk 3]
```diff
@@ -1,7 +1,26 @@
...
+   ResetColor();  
+   total += data.score;
+}
+WriteLine ("-----");
+WriteLine ($"{total,4}    Total");
+//Method to chgeck if the word is valid
+bool isValid (string word) => word.Length >= 4 && word.Contains (letters[0]) && word.All (letters.Contains);
```
Comment (rajeevramanTMM) [inline Training/Program.cs:20]: Provide method name in capital and leave one line space in between.

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Training/Program.cs (hunk 3 above)
Comment (Ajay07Metamation) [reply Training/Program.cs]: All changes done in the latest commit

**Pull request closed without merging**